setup.py

```diff
@@ -1,8 +1,13 @@
+from os import sys, path
+
 from skbuild import setup
 
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from itkVersion import VERSION
```

Comment:
Ah, yes. I assumed that setuptools automatically populated `<package>.__version__` based on the `version=` argument to `setup`.